### ISSUE TITLE
libp2p migration: cannot rename peerstore dir

### ISSUE THREAD

**Issue by twoeths** [ChainSafe/lodestar#3847]
**Describe the bug**

Got this error after we migrate libp2p

```
Mar 11 03:12:26 contabo-20 systemd[1]: Started Lodestar beacon.
Mar 11 03:12:27 contabo-20 beacon_run.sh[2539444]: Debugger listening on ws://0.0.0.0:9229/4334cbb4-080c-4877-912a-896fb09d0b70
Mar 11 03:12:27 contabo-20 beacon_run.sh[2539444]: For help, see: https://nodejs.org/en/docs/inspector
Mar 11 03:12:29 contabo-20 beacon_run.sh[2539444]: Mar-11 03:12:29.347[]                 info: Lodestar version=v0.36.0/dapplion/network-globals//5d37d6f5 (git), network=prater
Mar 11 03:12:29 contabo-20 beacon_run.sh[2539444]: Mar-11 03:12:29.356[]                 info: Connected to LevelDB database name=/home/devops/beacon/peerstore
Mar 11 03:12:29 contabo-20 beacon_run.sh[2539444]:  ✖ Error: ENOTEMPTY: directory not empty, rename '/home/devops/beacon/peerstore' -> '/home/devops/beacon/peerstore.pre-0.36.0.bak'
Mar 11 03:12:29 contabo-20 beacon_run.sh[2539444]:     at Object.renameSync (node:fs:980:3)
Mar 11 03:12:29 contabo-20 beacon_run.sh[2539444]:     at deleteOldPeerstorePreV036 (/usr/src/lodestar/packages/cli/src/migrations/deleteOldPeerstore.ts:35:8)
Mar 11 03:12:29 contabo-20 beacon_run.sh[2539444]:     at Object.beaconHandler [as handler] (/usr/src/lodestar/packages/cli/src/cmds/beacon/handler.ts:60:3)
Mar 11 03:12:29 contabo-20 systemd[1]: beacon.service: Main process exited, code=exited, status=1/FAILURE
Mar 11 03:12:29 contabo-20 systemd[1]: beacon.service: Failed with result 'exit-code'.
Mar 11 03:12:29 contabo-20 systemd[1]: beacon.service: Scheduled restart job, restart counter is at 79.
Mar 11 03:12:29 contabo-20 systemd[1]: Stopped Lodestar beacon.
```

**Expected behavior**

No error

**Comment by dapplion** [comment]
Seems that the folder may contain a DELETE file that only affects MacOS users https://stackoverflow.com/questions/35528425/enotempty-directory-not-empty-rename-javascript-npm-ubuntu-serve

@tuyennhv did you run this on ubuntu or your local env?

**Comment by twoeths** [comment]
> @tuyennhv did you run this on ubuntu or your local env?

I run it in contabo-20

**Comment by dapplion** [comment]
What about attempting to rename, else remove completely and continue?

**Comment by wemeetagain** [comment]
Should be resolved with #3915
